misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractCharSequenceExpectationsTest.kt

```diff
@@ -18,6 +20,8 @@ abstract class AbstractCharSequenceExpectationsTest(
     private val notToEndWithSpec: Fun1<CharSequence, CharSequence>,
     private val toMatchSpec: Fun1<CharSequence, Regex>,
     private val notToMatchSpec: Fun1<CharSequence, Regex>,
+    private val lengthFeatureSpec: Feature0<CharSequence, Int>,
+    private val lengthSpec: Feature1<CharSequence, Expect<Int>.() -> Unit, Int>,
```

Comment:
the previous definition was correct:
```suggestion
    private val lengthSpec: Fun1<CharSequence, Expect<Int>.() -> Unit>
```

Reply:
@robstoll - Should this be the constructor's signature?

```
abstract class AbstractCharSequenceExpectationsTest(
    private val toBeEmptySpec: Fun0<CharSequence>,
    private val notToBeEmptySpec: Fun0<CharSequence>,
    private val notToBeBlankSpec: Fun0<CharSequence>,
    private val toStartWithSpec: Fun1<CharSequence, CharSequence>,
    private val notToStartWithSpec: Fun1<CharSequence, CharSequence>,
    private val toEndWithSpec: Fun1<CharSequence, CharSequence>,
    private val notToEndWithSpec: Fun1<CharSequence, CharSequence>,
    private val toMatchSpec: Fun1<CharSequence, Regex>,
    private val notToMatchSpec: Fun1<CharSequence, Regex>,
    private val lengthFeatureSpec: Feature0<CharSequence, Int>,
    private val lengthSpec: Fun1<CharSequence, Expect<Int>.() -> Unit>
) : ExpectationFunctionBaseTest() {
```